deploy/gk-deploy

```diff
@@ -420,6 +420,7 @@ if [[ ${LOAD} -eq 0 ]]; then
       eval_output "${CLI} create -f ${TEMPLATES}/glusterfs-template.yaml 2>&1"
     fi
     eval_output "${CLI} policy add-role-to-user edit system:serviceaccount:${NAMESPACE}:heketi-service-account 2>&1"
+    eval_output "${CLI} adm policy add-scc-to-user privileged -z heketi-service-account"
```

Comment:
Please look at line 314, we have that as a requirement. Should we leave it as a requirement instead of doing it in script?